i3_resurrect/main.py

```diff
@@ -185,13 +185,16 @@ def remove(workspace, directory, profile, target):
     """
     Remove saved layout or programs.
     """
+    # Filter title string for saving in filename
+    workspace_id = util.filename_filter(workspace)
```

Comment:
Seeing as this is only used in the scope of the `elif` block below, maybe move it down there (line 196)?

Reply:
ok